federation/cmd/federation-apiserver/app/core.go

```diff
@@ -18,6 +18,14 @@ package app
 
 import (
 	"github.com/golang/glog"
+
+	// HACK to ensure that rest mapper from pkg/api is registered for groupName="".
+	// This is required because both pkg/api/install and federation/apis/core/install
+	// are installing their respective groupMeta at the same groupName.
+	// federation/apis/core/install has only a subset of resources and hence if it gets registered first, then installation of v1 API fails in pkg/master.
+	// TODO(nikhiljindal): Fix this by ensuring that pkg/api/install and federation/apis/core/install do not conflict with each other.
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
> I think this will break the swagger spec that federation puts out? Which may not be a concern?

btw, I just checked, while federation-apiserver imports pkg/api/install, the swagger spec for federation-apiserver does not include all api/v1 resources, only the ones registered in federation/apis/core/install.